src/test/java/seedu/address/logic/parser/TagCommandParserTest.java

```diff
@@ -39,12 +39,15 @@ public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, "a t/colleague", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 TagCommand.MESSAGE_USAGE));
 
+        // Index missing
+        assertParseFailure(parser, "t/colleague t/gym", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                TagCommand.MESSAGE_USAGE));
+
         // Missing tags (no tags specified)
         assertParseFailure(parser, "1", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 TagCommand.MESSAGE_USAGE));
 
-        // Index missing
-        assertParseFailure(parser, "t/colleague t/gym", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                TagCommand.MESSAGE_USAGE));
+        // Tags not in alphanumeric and space format
```

Comment:
Maybe can make this comment clearer, something like "Tag descriptions contain non-alphanumeric or space characters", just to make it clear that it's referring to the tag's description and what the format is?

Reply:
Thats sounds clearer, i'll make the changes thanks